src/wp-includes/media.php

```diff
@@ -5794,6 +5796,14 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 		if ( wp_lazy_loading_enabled( $tag_name, $context ) ) {
 			$loading_attrs['loading'] = 'lazy';
 		}
+
+		// Do not override `decoding` if it is already set.
+		if ( 'img' === $tag_name ) {
+			$loading_attrs['decoding'] = 'async';
+			if ( array_key_exists( 'decoding', $attr ) ) {
+				$loading_attrs['decoding'] = $attr['decoding'];
+			}
+		}
```

Comment:
I don't think this is needed? There's already the logic above to add the attribute, this seems like an unnecessary duplicate.

Reply:
True. I remember this was removed before, but I missed removing it again when resolving some merge conflicts.